fiberplane-models/src/integrations.rs

```diff
@@ -0,0 +1,55 @@
+use crate::timestamps::Timestamp;
+#[cfg(feature = "fp-bindgen")]
+use fp_bindgen::prelude::Serializable;
+use serde::{Deserialize, Serialize};
+use strum_macros::{Display, EnumIter};
+use typed_builder::TypedBuilder;
+
+#[derive(Clone, Debug, Deserialize, PartialEq, Eq, Serialize, TypedBuilder)]
+#[cfg_attr(
+    feature = "fp-bindgen",
+    derive(Serializable),
+    fp(rust_module = "fiberplane_models::integrations")
+)]
+#[non_exhaustive]
+#[serde(rename_all = "camelCase")]
+pub struct Integration {
+    pub id: IntegrationId,
+    pub status: IntegrationStatus,
+
+    #[builder(setter(into))]
+    pub created_at: Timestamp,
+    #[builder(setter(into))]
+    pub updated_at: Timestamp,
+
+    #[builder(default, setter(strip_option, into))]
+    #[serde(default, skip_serializing_if = "Option::is_none")]
+    pub data: Option<serde_json::Value>,
```

Comment:
```suggestion
    pub data: Option<Value>,
```

We need to import value at the top level otherwise fp-bindgen will generate this json:
```
    data?: serde_json::Value;
```

Reply:
nice catch! but i yeeted `data` off the struct anyways so should be good now